releasenotes/notes/sparselabelop-class-57ffd57121dadfe5.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds :class:`.SparseLabelOp` which maps sparse labels to coefficient values.
```

Comment:
Same here. Its a single line item that of course they should be able to click and read the class doc but maybe it can say a little more - it seems kinda sparse as a description!